rofeeah_ayeni/alpha_challenges/the_basics.md

```diff
@@ -48,9 +48,9 @@ The first statement is assigning weather to the value of "cloudy" string. The se
 x = 2
 
 if x == 3
-  puts "sushi is tasty"
+	puts "sushi is tasty"
```

Comment:
Hmm...looks like the original tabbing was affected by whatever change you made. You want to make sure that you're indenting using spaces with a tab width of 2.